### ISSUE TITLE
Mobile Support

### ISSUE THREAD

**Issue by kitling** [SpongePowered/Ore#37]
When viewing ore-staging on my iPhone, the interface looks fine (but tiny) until I start zooming in (because it looks tiny). At that point, the main page and the title bar scale seperately and start overlapping each other.

**Comment by gratimax** [comment]
I definitely acknowledge this as a problem, and I think that starting out with using Bootstrap was a step in the right direction. I think that it is certainly possible to achieve a perfectly responsive Ore website that works well with mobile phones, but that is simply not the most pressing issue. As I hope you can understand, Ore is designed to be used primarily on the desktop. As the more urgent issues get fixed and Ore gets more complete we will certainly look into this kind of issue.

**Comment by kitling** [comment]
I completely agree, but got to document the issue, or it may not ever come to someone's attension. :)

**Comment by Calypo** [comment]
I'm thinking that using Foundation would be a better idea. It's more lightweight and essentially more flexible.

**Comment by gratimax** [comment]
@Calypo Perhaps when we started this project Foundation would have been a better idea, but now the overhead in converting from one frontend framework to another makes me think that it is not worth it. I personally don't see any big difference between Foundation and Bootstrap short of Bootstrap having 60,000 more stars on Github...
